src/plugins/discover/public/services/doc_views/components/doc_viewer_table/table.tsx

```diff
@@ -157,7 +158,12 @@ export const DocViewerTable = ({
     (field: string) => {
       const fieldMapping = mapping(field);
       const displayName = fieldMapping?.displayName ?? field;
-      const fieldType = isNestedFieldParent(field, dataView) ? 'nested' : fieldMapping?.type;
+      // const fieldType = isNestedFieldParent(field, dataView) ? 'nested' : fieldMapping?.type;
```

Comment:
this seems to be a leftover that can be cleaned

Reply:
Apparently, I still need to learn to clean up after myself!